src/molecules/Footer/Top.tsx

```diff
@@ -40,7 +40,32 @@ export const Subtitle = styled(H5)`
 export const CityInputWrapper = styled.div`
   width: 100%;
   max-width: 555px;
+  margin-bottom: ${({ theme }) => theme.ruler[2]}px;
+`;
+
+export const CityContainer = styled.div`
   margin-bottom: ${({ theme }) => theme.ruler[16]}px;
```

Comment:
so this is actually `16 * 4` for the spacing as the ruler is based on 4. which would make this 64px. is that what you want?

Reply:
I just moved this margin-bottom down from `CityInputWrapper`, so it's the same padding originally used, just instead of between the input field and the section, it's between the (input field first), city list and the section now.

Hopefully that makes sense, hard to explain without a diagram 😸.

Reply:
No that makes sense! :)